app/css/detail.scss

```diff
@@ -105,6 +105,10 @@
     span {
       color: #777;
     }
+    // fix for multiple phone numbers
```

Comment:
I don't think we need a comment here; this change shouldn't surprise people down the road who look at this code.